extensions/amp-google-assistant-assistjs/validator-amp-google-assistant-assistjs.protoascii

```diff
@@ -0,0 +1,84 @@
+#
+# Copyright 2021 The AMP HTML Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS-IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the license.
+#
+
+tags: {  # amp-google-assistant-assistjs
+  html_format: AMP
+  tag_name: "SCRIPT"
+  extension_spec: {
+    name: "amp-google-assistant-assistjs"
+    version: "0.1"
+    version: "latest"
+  }
+  attr_lists: "common-extension-attrs"
+}
+tags: {  # <amp-google-assistant-voice-button>
+  html_format: AMP
+  tag_name: "AMP-GOOGLE-ASSISTANT-VOICE-BUTTON"
+  requires_extension: "amp-google-assistant-assistjs"
+  attrs: {
+    name: 'project';
+    mandatory: true;
+  }
+  attrs: {
+    name: 'dev';
+  }
+  attrs: {
+    name: 'server'
+  }
+  attr_lists: "extended-amp-global"
+  amp_layout: {
+    supported_layouts: RESPONSIVE
+  }
+}
+tags: {  # <amp-google-assistant-voice-bar>
+  html_format: AMP
+  tag_name: "AMP-GOOGLE-ASSISTANT-VOICE-BAR"
+  requires_extension: "amp-google-assistant-assistjs"
+  attrs: {
+    name: 'project';
+    mandatory: true;
```

Comment:
remove the trailing `;` on this and other lines.

Reply:
Removed all attrs config since we don't need any attributes in all custom elements right now.